openpype/hosts/maya/startup/userSetup.py

```diff
@@ -13,23 +12,40 @@
 
 print("Starting OpenPype usersetup...")
 
+settings = get_project_settings(os.environ['AVALON_PROJECT'])
+
+# Loading plugins explicitly.
+if settings["maya"]["explicit_plugins_loading"]["enabled"]:
+    def _explicit_load_plugins():
+        project_settings = get_project_settings(os.environ["AVALON_PROJECT"])
+        maya_settings = project_settings["maya"]
+        explicit_plugins_loading = maya_settings["explicit_plugins_loading"]
+        if explicit_plugins_loading["enabled"]:
+            for plugin in explicit_plugins_loading["plugins_to_load"]:
+                if plugin["enabled"]:
+                    print("Loading " + plugin["name"])
+                    try:
+                        cmds.loadPlugin(plugin["name"], quiet=True)
+                    except RuntimeError as e:
+                        print(e)
+
+    cmds.evalDeferred(
+        _explicit_load_plugins,
+        lowestPriority=True
+    )
 
 # Open Workfile Post Initialization.
 key = "OPENPYPE_OPEN_WORKFILE_POST_INITIALIZATION"
 if bool(int(os.environ.get(key, "0"))):
+    def _log_and_open():
+        print("Opening \"{}\"".format(os.environ["AVALON_LAST_WORKFILE"]))
+        cmds.file(os.environ["AVALON_LAST_WORKFILE"], open=True, force=True)
```

Comment:
Would this be more readable like this?
```suggestion
        path = os.environ["AVALON_LAST_WORKFILE"]
        print("Opening \"{}\"".format(path))
        cmds.file(path, open=True, force=True)
```